featuretools/synthesis/dfs.py

```diff
@@ -174,6 +175,15 @@ def dfs(entities=None,
     if not isinstance(entityset, EntitySet):
         entityset = EntitySet("dfs", entities, relationships)
 
+    for entry_point in pkg_resources.iter_entry_points('featuretools_dfs'):
```

Comment:
add a comment to this block? something like

"Call functions registered by other libraries with DFS arguments"

Reply:
Added